sdk/tables/data-tables/README.md

```diff
@@ -108,28 +108,30 @@ const AzureTables = require("@azure/data-tables");
 Alternatively, selectively import only the types you need:
 
 ```javascript
-const { TableServiceClient, TablesSharedKeyCredential } = require("@azure/data-tables");
+const { TableServiceClient } = require("@azure/data-tables");
+const { AzureNamedKeyCredential } = require("@azure/core-auth");
```

Comment:
In other libraries, we would export the credential type from the library, so customers did not need to utter a reference to `@azure/core-auth` (for example, this is common with `AzureKeyCredential`.  Should we do the same here?

Reply:
Yeah that makes sense!